BoincRpc/BoincRpc.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Is there any advantage of specifying a higher .NET Standard version? If I am not mistaken, the guideline is to choose the lowest version that still supports all features required by the library. This ensures that the library can be used with as many frameworks as possible.